GoMain/src/main/main.go

```diff
@@ -60,9 +66,13 @@ const (
 	certificateFilePath    = edgeDir + "/data/cert"
 	containerWhiteListPath = edgeDir + "/data/cwl"
 	passPhraseJWTPath      = edgeDir + "/data/jwt"
+	binPath                = "GoMain/bin"
```

Comment:
What do you think of using edgeDir as below?
```suggestion
	binPath                = edgeDir + "/bin"
```
If this is more reasonable, the YamlFileName and the ConfigFileName are should be changed as well.

Reply:
Ok will consider for next PR

Reply:
Created the new issue #136. ^^